jonprl-mode.el

```diff
@@ -48,6 +48,28 @@
   :type 'file
   :group 'jonprl)
 
+(defcustom jonprl-pretty-symbols
+  '(("*" . ?×)
+   ("->" . ?→)
+   (":" . ?∶)
+   ("<" . ?⟨)
+   ("<>" . ?⋄)
+   ("=>" . ?⇒)
+   ("=def=" . ?≝)
+   (">" . ?⟩)
```

Comment:
Same problem here, since this symbol is often followed by a semicolon, which prevents pretty-symbols-mode from rendering it properly.